modules/monitoring-disabling-grafana.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+[id="disabling_grafana_{context}"]
+= Disabling the default Grafana deployment
+
+By default a read-only Grafana instance is deployed with a collection of dashboards displaying cluster metrics. This Grafana instance is not user-configurable.
+
+You can disable this Grafana deployment, causing the associated resources to be deleted from the cluster. You might do this if you don't need these dashboards and want to conserve resources in your cluster. You will still be able to graph metrics and view the dashboards included in the OpenShift Container Platform web console. Grafana can be safely enabled again at any time.
+
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created the `cluster-monitoring-config` `ConfigMap` object.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Edit the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. Add `enabled: false` for the `grafana` component under `data/config.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    grafana:
+      enabled: false
+----
+
+. Save the file to apply the changes. The resources will begin to be removed automatically when you apply the change.
++
+[WARNING]
+====
+This change results in some components, including Prometheus, being restarted. This may lead to previously collected metrics being lost if you have not followed the steps in xref:../monitoring/configuring-the-monitoring-stack.html#configuring-persistent-storage[Configuring persistent storage].
+====
+
+. Check that the pods for Grafana are no longer running. The following example lists the status of pods in the `openshift-monitoring` project:
```

Comment:
Check that the pods for Grafana are no longer running
->
Check the Grafana pods are no longer running

Reply:
Fixed.